recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -30,7 +30,7 @@ PACKAGECONFIG[encryptedmedia] = "-DENABLE_ENCRYPTED_MEDIA=ON,-DENABLE_ENCRYPTED_
 PACKAGECONFIG[fetchapi] = "-DENABLE_FETCH_API=ON,-DENABLE_FETCH_API=OFF,"
 PACKAGECONFIG[gamepad] = "-DENABLE_GAMEPAD=ON,-DENABLE_GAMEPAD=OFF,"
 PACKAGECONFIG[geolocation] = "-DENABLE_GEOLOCATION=ON,-DENABLE_GEOLOCATION=OFF,geoclue"
-PACKAGECONFIG[gst_gl] = "-DUSE_GSTREAMER_GL=ON,-DUSE_GSTREAMER_GL=OFF,gstreamer1.0-plugins-base"
+PACKAGECONFIG[gst_gl] = "-DUSE_GSTREAMER_GL=ON,-DUSE_GSTREAMER_GL=OFF,gstreamer1.0-plugins-base wpebackend-fdo"
```

Comment:
This is not correct, I think a better approach would to add a new PACKAGECONFIG for `USE_WPE_VIDEO_PLANE_DISPLAY_DMABUF` which would then depend on wpebackend-fdo. The gst-gl support has no dependency on -fdo per se, AFAIK.

Reply:
Actually it seems the wpebackend-fdo library is need always:

```
| CMake Error in Source/WebCore/CMakeLists.txt:
|   Found relative path while evaluating include directories of "WebCore":
| 
|     "WPEBACKEND_FDO_INCLUDE_DIRS-NOTFOUND"
| 
| 
| 
| CMake Error in Source/WebCore/CMakeLists.txt:
|   Found relative path while evaluating include directories of
|   "WebCoreTestSupport":
| 
|     "WPEBACKEND_FDO_INCLUDE_DIRS-NOTFOUND"
| 
| 
| 

```

```
3fc2b8cee129b (mcatanzaro@igalia.com   2018-01-17 23:18:17 +0000 66) list(APPEND WebCore_SYSTEM_INCLUDE_DIRECTORIES
fdfb04c888295 (carlosgc@webkit.org     2019-05-21 12:45:08 +0000 67)     ${ATK_INCLUDE_DIRS}
91f235214312c (carlosgc@webkit.org     2017-07-12 06:59:15 +0000 68)     ${GIO_UNIX_INCLUDE_DIRS}
75ce5873e3fcd (zandobersek@gmail.com   2017-05-09 11:07:57 +0000 69)     ${GLIB_INCLUDE_DIRS}
34e2b42c073d7 (commit-queue@webkit.org 2017-12-20 01:10:11 +0000 70)     ${LIBTASN1_INCLUDE_DIRS}
4862022695e7c (gns@gnome.org           2017-10-04 07:46:25 +0000 71)     ${UPOWERGLIB_INCLUDE_DIRS}
c7d6af038dcbf (philn@webkit.org        2020-01-16 15:57:34 +0000 72)     ${WPEBACKEND_FDO_INCLUDE_DIRS} <<<<
```

Reply:
I thought that var would expand to empty string if the lib is not found. my mistake then, this cmake file needs to be fixed :) Hacking the OE recipe is not the right approach I'm afraid.

Reply:
Please test https://bugs.webkit.org/show_bug.cgi?id=219835 :)

Reply:
Commented in the upstream bug: https://bugs.webkit.org/show_bug.cgi?id=219835#c2

Reply:
The fix for https://bugs.webkit.org/show_bug.cgi?id=219835 also fix this. Closing this issue. Thanks.